examples/faith/pegasus_workflow_submit.sh

```diff
@@ -0,0 +1,3 @@
+cd output
+
+pycbc_submit_dax
```

Comment:
Do we want to add this to the automated github workflows that run with every commit? If so the workflow example must be setup to run very quickly (ie. in under 10 minutes with no more than 2 active condor jobs).